### PR TITLE
BaSD-MS-45: Create Clients View

### PR DESCRIPTION
**Pull request by ripellizzari**
Fully connected and in interaction with the back end.

Show list of all the open Clients.
Open one to see the detail.
Modify a client.
Add a new one.
Delete a client.

### CODE REVIEW

File: public/js/clientLIst.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+};
+
+const openNewClient = () => {
+  window.location.href = `${window.location.origin}/views/clientForm.html`;
+};
+
+const deleteClient = (id, name, event) => {
+  event.stopPropagation();
+  document.getElementById('modal-title').innerText = 'You are about to delete';
+  document.getElementById('modal-data-inputs').innerText = `${name}`;
```
Comment (estefrare) [inline public/js/clientLIst.js]: no need to use string template here
```suggestion
  document.getElementById('modal-data-inputs').innerText = name;
```

File: public/views/clientForm.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position List</title>
```
Comment (estefrare) [inline public/views/clientForm.html]: update title

File: public/views/clientForm.html [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/styles.css">
+    <link rel="stylesheet" href="../styles/modal.css">
+    <script src="../js/clientForm.js"></script>
```
Comment (estefrare) [inline public/views/clientForm.html]: add `async` attribute and remove the `window.addEventListener('load', modalOnLoad);` on the js file

File: public/views/clientList.html [hunk 4]
```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Position List</title>
```
Comment (estefrare) [inline public/views/clientList.html]: update title

File: public/views/clientList.html [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+              <button id="cancel-button">CANCEL</button>
+              <button id="procced-button">CONFIRM</button>
+            </div>
+        </div>
+      </section>
+  <footer>
+    <div>© Quito's group, Radium Rocket "Become a Software Developer 2021"</div>
+  </footer>
+  <script type="text/javascript" src="../js/modal.js"></script>
+    <script type="text/javascript" src="../js/clientList.js"></script>
```
Comment (estefrare) [inline public/views/clientList.html]: put the `<script>` into the `<head>` tag at the top and add `async` attribute

**Review by estefrare — CHANGES_REQUESTED** [review]

File: public/js/clientLIst.js (hunk 1 above)
Comment (ripellizzari) [reply public/js/clientLIst.js]: done!

File: public/views/clientForm.html (hunk 2 above)
Comment (ripellizzari) [reply public/views/clientForm.html]: done!

File: public/views/clientForm.html (hunk 3 above)
Comment (ripellizzari) [reply public/views/clientForm.html]: done!

File: public/views/clientList.html (hunk 4 above)
Comment (ripellizzari) [reply public/views/clientList.html]: done!

File: public/views/clientList.html (hunk 5 above)
Comment (ripellizzari) [reply public/views/clientList.html]: Done!

File: public/js/clientLIst.js [hunk 6]
```diff
@@ -0,0 +1,71 @@
+const tableContent = document.getElementById('clients-table-content');
+
```
Comment (estefrare) [inline public/js/clientLIst.js:2]: rename this file to `public/js/clientList.js` (The second letter of `List` is capitalized and should not)

**Review by estefrare — CHANGES_REQUESTED** [review]

**Review by estefrare — APPROVED** [review]

**Pull request merged**